Sources/GRPCProtobufCodeGen/ProtobufCodeGenParser.swift

```diff
@@ -120,10 +120,14 @@ extension CodeGenerationRequest.ServiceDescriptor {
       generatedUpperCase: NamingUtils.toUpperCamelCase(descriptor.name),
       generatedLowerCase: NamingUtils.toLowerCamelCase(descriptor.name)
     )
+
+    // Packages that are based on the path of the '.proto' file usually
+    // contain dots. For example: "grpc.test".
+    let cleanNamespace = package.replacingOccurrences(of: ".", with: "_")
     let namespace = CodeGenerationRequest.Name(
-      base: package,
-      generatedUpperCase: NamingUtils.toUpperCamelCase(package),
-      generatedLowerCase: NamingUtils.toLowerCamelCase(package)
+      base: cleanNamespace,
+      generatedUpperCase: NamingUtils.toUpperCamelCase(cleanNamespace),
+      generatedLowerCase: NamingUtils.toLowerCamelCase(cleanNamespace)
```

Comment:
The 'toUpperCamelCase()' and 'toLowerCamelCase()' methods are also transforming the snake case into camel case. This is why the generated type names (generated through CodeGenLib) don't contain underscores in the namespace prefix. On the other hand, the Input and Output types contain underscores in the namespace prefix. The same is true for the protocol names in protocol names in v1. This inconsistency is generated by the different SwiftProtobuf naming functions (some public and some internal that I can't use).

Reply:
I will update this PR after we merge the enum name PR then